transport/transport-runtime/transport-runtime.gradle

```diff
@@ -47,6 +79,11 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
     sourceSets {
+        main {
+            proto {
+                srcDir 'src/main/proto'
```

Comment:
why is this needed? afaict it should be the default even if you don't specify it

Reply:
Adding the above line will stop the android studio compiling about the syntax error.

Reply:
sgtm